server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/CloudFoundryLifecycle.java

```diff
@@ -26,12 +32,29 @@ public boolean checkModel() {
     @Override
     public void prepare() {
         //throw new UnsupportedOperationException();
+
+        Map<String, String> properties = context.getProperties().getPropertyValues();
+
+        if (!properties.isEmpty()) {
+            String username = properties.get("username");
```

Comment:
Just a small note: the Property api does not yet have a getPropertyValue operation. Thats why this way is the only option. I added a method to get a value in the Docker Image Push branch. Once that is merged you will have access to that